### PR TITLE
Search with fixed height and scrolling

### PR DESCRIPTION
**Pull request by uplne**
This PR is part of the sticky header so it is behind the zero audience AB test for internal testing only.

Sticky header requires search to have fixed heigh and user can scroll through search results.
Also when user starts scrolling the search window will close on desktop. It doesn't need to close on mobile as the search + header is taking over whole screen.

hwcb0gxdx7

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/navigation/search.js [hunk 1]
```diff
@@ -47,8 +49,31 @@ define([
...
                 $input.focus();
+
+                if (ab.shouldRunTest('Viewability', 'variant') && config.page.contentType !== 'Interactive') {
+                    checkInterval = setInterval(self.checkResults, 250);
+                }
+            }
+        };
+
+        // Check if google returned results as there is no callback from google API v2 for this
+        this.checkResults = function () {
```
Comment (sndrs) [inline static/src/javascripts/projects/common/modules/navigation/search.js]: this could run indefinitely couldn't it? feels like it should have a limit?

Comment (uplne) [inline static/src/javascripts/projects/common/modules/navigation/search.js]: Good point. I should clear it if user closes the search without searching

Comment (uplne) [inline static/src/javascripts/projects/common/modules/navigation/search.js]: I've updated this to make sure that it clears interval on each search toggle.

**Comment by Calanthe** [comment]
It looks ok to me :+1:

**Pull request merged**